servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -539,4 +824,38 @@ public void updatePinnedInodes(Set<Integer> inodes) {
       mPinnedInodes.addAll(Preconditions.checkNotNull(inodes));
     }
   }
+
+  /**
+   * A wrapper on necessary info after a move block operation
+   */
+  private static class MoveBlockResult {
+    private final boolean mDone;
+    private final long mBlockSize;
+    private final BlockStoreLocation mSrcLocation;
+    private final BlockStoreLocation mDstLocation;
+
+    MoveBlockResult(boolean done, long blockSize, BlockStoreLocation srcLocation,
+        BlockStoreLocation dstLocation) {
+      mDone = done;
+      mBlockSize = blockSize;
+      mSrcLocation = srcLocation;
+      mDstLocation = dstLocation;
+    }
+
+    boolean done() {
```

Comment:
Is this basically success or not? I think "success()" (and mSuccess) would be a better name for it.